dashbuilder-client/dashbuilder-dataset-editor/src/main/java/org/dashbuilder/dataset/editor/client/screens/DataSetDefWizardScreen.java

```diff
@@ -216,10 +216,10 @@ void onSave(final DataSetDef dataSetDef, final String message) {
 
     RemoteCallback<Path> saveSuccessCallback = new RemoteCallback<Path>() {
         @Override public void callback(Path path) {
+            currentWorkflow.clear();
```

Comment:
@jhrcek Thanks for looking into this.

In order to enable logging on client side => http://docs.jboss.org/errai/4.0.0.Final/errai/reference/html/sid-74908675.html

I'll try to setup the catching & logging of uncaught exceptions in dashbuilder's showcase when I get a chance.